sklearn/tree/_tree.pyx

```diff
@@ -177,13 +261,18 @@ cdef class ClassificationCriterion(Criterion):
         self.label_count_right = NULL
         self.label_count_total = NULL
 
-        # Count labels for each output
```

Comment:
I actually find this previous comment more helpful that the verbose comments you added